pkg/rust/examples/hello_world.rs

```diff
@@ -54,7 +54,7 @@ mod hello_world {
         };
 
         let first_row = &rows[0];
-        let first_value = &first_row[0];
+        let first_value = first_row.iter().next().unwrap();
```

Comment:
Looks nice, but.. in our case it might be not a good option because Index trait can cause panic in our cases.
So, we are currently returning by wrapping `Option<>`.

Reply:
Index of `Vec` also causes panic.
So it looks fine.

Reply:
> Index of Vec also causes panic.

Ah, I think I need to mention the different point.
After we contain both `Vec` and `HashMap` in `Row`, then it will become simply not possible to support `Index` because of `HashMap`. so.. though we add `Index` trait, we may need to remove it in near future.

Reply:
`HashMap` also supports Index, but (https://doc.rust-lang.org/stable/std/collections/struct.HashMap.html#impl-Index%3C%26Q%3E-for-HashMap%3CK%2C%20V%2C%20S%3E)
I don't think it will be needed in the future.

Reply:
Oh, I didn't think about `Index` impl in `HashMap`.

Then, there can be something sounds interesting.
Let's assume.. that we have `Row` which contains both `Vec<Value>` and `HashMap<String, Value>`.
Simplifying.. e.g.
```rust
pub struct Row(Vec<Value>, HashMap<String, Value>)`
```

Then, can we impl `Index` for Row` which can scan both `Vec` and `HashMap`?

Reply:
You can scan all of them, and you can think of some function going into it.
Here is an example.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8fb4b39299d80030ca63379bc16d1317

Reply:
Thanks! Now I got it.

That becomes possible by declaring the custom key for `Index` trait!
I was stuck at.. how to merge both `usize` for `Vec` and `&str` for `HashMap` together.
Looks interesting..